server/config/types.ts

```diff
@@ -57,9 +57,11 @@ export const StatementStatus = toEnum({
 })
 
 export const RelocationType = toEnum({
-  SIDE: { value: 'SIDE', label: 'Side relocation' },
+  PRIMARY: { value: 'PRIMARY', label: 'Primary relocation' },
```

Comment:
can't rename this/remove SIDE as existing data will rely on it being present to pull back the description.

Might be worth have an active/inactive flag which doesn't get rendered when displaying the form but can be used by `toLabel`

Reply:
Implemented as suggested.